docs/ref/models/querysets.txt

```diff
@@ -580,6 +593,23 @@ A few subtleties that are worth mentioning:
 * Calling :meth:`only()` and :meth:`defer()` after ``values()`` doesn't make
   sense, so doing so will raise a ``NotImplementedError``.
 
+* Combining transforms and aggregates requires the use of two :meth:`annotate`
+  calls, either explicitly or as keyword arguments to :meth:`values`. As above,
+  if the transform has been registered on the relevant field type the first
+  :meth:`annotate` can be omitted, thus the following examples are equivalent::
+
+    >>> from django.db.models import CharField, Count
+    >>> from django.db.models.functions import Lower
+    >>> CharField.register_lookup(Lower, 'lower')
+    >>> Blog.objects.values('author__name__lower').annotate(entries=Count('entry'))
```

Comment:
I think these examples need correcting as `Blog` doesn't have an `author` field.

Reply:
That's a good point. I'd expanded on an example higher in the file, which was fixed in the mainline as #9601. I'll apply the same change here.